Decred Wallet/Features/Send/SendViewController.swift

```diff
@@ -369,30 +369,30 @@ extension SendViewController {
 
     func calculateAndDisplayUSDAmount() {
         guard let dcrAmount = Double(self.amountTextField.text ?? ""), let exchangeRate = self.exchangeRate else {
-            self.usdAmountLabel.text = "- USD"
+            self.usdAmountLabel.text = "0 USD"
```

Comment:
This would give the impression that the USD amount equivalent to the dcr amount in the amount text field is "0 USD" rather than unknown.